Makefile

```diff
@@ -49,6 +49,7 @@ help:
 	@echo "make generate - generate API in api/facades and docs from schema"
 	@echo "make update-schema - update schema with current juju develop"
 	@echo "make test - run unit tests"
+	@echo "make test-js - run js unit tests"
```

Comment:
Thanks! Also add lint-js while you are there?

Reply:
done